deploy/csi-rclone/templates/csi-rclone-storageclass.yaml

```diff
@@ -0,0 +1,9 @@
+apiVersion: storage.k8s.io/v1
+kind: StorageClass
+metadata:
+  name: {{ include "chart.fullname" . }}
```

Comment:
The storage class name should be hardcoded or in the values file. I see you already have the value for it in the values file.